### PR TITLE
Branch002

### PR DESCRIPTION
**Pull request by Poikka001**

### CODE REVIEW

File: src/main/java/mate/academy/intro/service/RoleService.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package mate.academy.intro.service;
+
+import mate.academy.intro.model.Role;
+
+public interface RoleService {
+    Role save(Role role);
+
+    boolean exists(Role.RoleName roleName);
```
[the remark this answers is not in the document]
Comment (Poikka001) [reply src/main/java/mate/academy/intro/service/RoleService.java:8]: sorry)

File: src/main/java/mate/academy/intro/config/SecurityConfig.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                .sessionManagement(manager ->
+                        manager.sessionCreationPolicy(SessionCreationPolicy.STATELESS))
+                .build();
+    }
+
+    @Bean
+    public PasswordEncoder passwordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
+}
```
Comment (kerrrusha) [inline src/main/java/mate/academy/intro/config/SecurityConfig.java:43]: need 1 blank line at the EOF

File: src/main/java/mate/academy/intro/repository/role/RoleRepository.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package mate.academy.intro.repository.role;
+
+import mate.academy.intro.model.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (kerrrusha) [inline src/main/java/mate/academy/intro/repository/role/RoleRepository.java:7]: it does nothing to interfaces
```suggestion
```

File: src/main/java/mate/academy/intro/exception/RegistrationException.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package mate.academy.intro.exception;
+
+public class RegistrationException extends Exception {
```
Comment (kerrrusha) [inline src/main/java/mate/academy/intro/exception/RegistrationException.java:3]: consider using unchecked exceptions
```suggestion
public class RegistrationException extends RuntimeException {
```

**Review by kerrrusha — CHANGES_REQUESTED** [review]

**Pull request merged**